tests/linearGaussian_snre_test.py

```diff
@@ -23,7 +23,7 @@
     VIPosterior,
     ratio_estimator_based_potential,
 )
-from sbi.inference.trainers.nre.nre_base import RatioEstimator
+from sbi.neural_nets.ratio_estimators import RatioEstimator
```

Comment:
good catch! this was not detected before because it was just used for typing I guess.